tools/docker/builder/openwrt/scripts/build-openwrt.sh

```diff
@@ -34,6 +35,23 @@ case $TARGET_DEVICE in
             echo "CONFIG_TARGET_${TARGET_SYSTEM}_${SUBTARGET}=y"
             echo "CONFIG_TARGET_${TARGET_SYSTEM}_${SUBTARGET}_${TARGET_PROFILE}=y"
             echo "CONFIG_PACKAGE_prplmesh${PRPLMESH_VARIANT}=y"
+            # AMBIORIX modules to work with u-bus
+            echo "CONFIG_PACKAGE_amxb-ubus=y"
+            # AMBIORIX applications for debug
+            echo "CONFIG_PACKAGE_amxb-inspect=y"
+            echo "CONFIG_PACKAGE_amxo-cg=y"
+            echo "CONFIG_PACKAGE_amxrt=y"
+            # AMBIORIX libs
+            echo "CONFIG_PACKAGE_libamxb=y"
+            echo "CONFIG_PACKAGE_libamxc=y"
+            echo "CONFIG_PACKAGE_libamxd=y"
+            echo "CONFIG_PACKAGE_libamxj=y"
+            echo "CONFIG_PACKAGE_libamxo=y"
```

Comment:
You might also want to check if all the config lines are needed, or if some are depended on by others and can be removed (maybe you did already).